build.ps1

```diff
@@ -34,15 +33,32 @@ function ExecuteWithRetry($Command) {
     }
 }
 
-ExecuteWithRetry docker pull $ImageBuilderImageName
+function Invoke-CleanupDocker()
+{
+    if ($CleanupDocker) {
+        docker system prune -a -f
+    }
+}
 
-& docker run --rm `
-    -v /var/run/docker.sock:/var/run/docker.sock `
-    -v "${PSScriptRoot}:/repo" `
-    -w /repo `
-    $ImageBuilderImageName `
-    build --manifest "manifest.json" --path "$DockerfilePath" "$ImageBuilderCustomArgs"
+Invoke-CleanupDocker
 
-if ($LastExitCode -ne 0) {
-    throw "Failed executing ImageBuilder."
+try {
+    ExecuteWithRetry docker pull 'microsoft/dotnet-buildtools-prereqs:image-builder-jessie-20180117125404'
```

Comment:
We can't have the image-builder name duplicated in two spots.  This is going to get out of sync with the Dockerfile.